deps/build.jl

```diff
@@ -96,14 +96,20 @@ If you already have a polymake installation you need to set the environment vari
         using Pkg: depots1
         function prepare_env()
             ENV["PERL5LIB"]=$perllib
-            ENV["POLYMAKE_USER_DIR"] = abspath(joinpath(depots1(),"polymake_user"))
+            user_dir = ENV["POLYMAKE_USER_DIR"] = abspath(joinpath(depots1(),"polymake_user"))
+            if Base.Filesystem.isdir(user_dir)
+                del = filter(i -> Base.Filesystem.is_dir(i) && startswith(i, "wrappers."), readdir(user_dir))
```

Comment:
This is probably `isdir` and not `is_dir`

Reply:
Indeed. Fixed.